server/tests/models/device_spec.js

```diff
@@ -0,0 +1,60 @@
+import chai, { expect } from 'chai';
+import Device from '../../models/device';
+
+chai.config.incldueStack = true;
+
+describe('# Device Model', () => {
+  before(() => {
+    Device.remove({}, () =>{});
+  });
+
+  after(() => {
+    Device.remove({}, () =>{});
+  });
+
+  context('given a valid device', () => {
+    let device = new Device({
+      name: 'My Phone',
+      type: 'email',
+      address: '555-555-5555'
+    });
+
+    it('creates a devices', (done) => {
+      Device.create(device, (err, d) => {
+        expect(err).to.not.exist;
+
+        expect(d.name).to.equal(device.name);
+        expect(d.type).to.equal(device.type);
+        expect(d.address).to.equal(device.address);
+        done();
+      });
+    });
+  });
+  context('when the type is not an enum', () => {
```

Comment:
After some of these contexts there's a new line, and after others there isn't. I'd prefer a new line after each context, but whichever makes it consistent is fine.